scripts/valhalla_build_config

```diff
@@ -61,7 +61,10 @@ config = {
     }
   },
   'additional_data': {
-    'elevation': '/data/valhalla/elevation/'
+    'elevation': '/data/valhalla/elevation/',
+    'elevation_url': Optional(str),
+    'elevation_url_gz': Optional(bool),
+    'elevation_dir': Optional(bool)
```

Comment:
isnt this redundant with the existing directory above? i agree the naming is unfortunate and it would be nice to be the same like `tile_dir` is but still we already have this so sadly i think we have to stick with just `elevation` rather than duplicate the idea

Reply:
yeah, i decided to bring this as a proposal. i will remove "elevation_dir" and leave only "elevation" then.